tests/acceptance-bdd/test_literature_suggestion_form.py

```diff
@@ -0,0 +1,392 @@
+
+from selenium.common.exceptions import (
+    ElementNotVisibleException,
+    WebDriverException,
+)
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import Select, WebDriverWait
+
+
+@scenario('literature_suggestion.feature', 'Submit article and verify record detail in the list page')
+def test_verify_article_record_in_the_list_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Submit article and verify record detail in the detail page')
+def test_verify_article_record_in_the_detail_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Accept article and verify the confirmation message')
+def test_verify_accepted_article():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Use the autocomplete for the supervisor institution')
+def test_autocomplete_supervisor_institution():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Use the autocomplete for the journal title')
+def test_autocomplete_journal_title():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Use the autocomplete for the conference title')
+def test_autocomplete_conference_title():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Use the autocomplete for the affilitaion')
+def test_autocomplete_affilation():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Import an article using the arXiv id')
+def test_import_from_arxiv():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Import an article using the DOI id')
+def test_import_from_doi():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing the wrong arXiv id')
+def test_wrong_arxiv():
+    pass
+
+
+def test_correct_arxiv():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing the wrong DOI id')
+def test_wrong_doi():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing the correct DOI id')
+def test_correct_doi():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a wrong url')
+def test_wrong_url():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a wrong thesis date')
+def test_wrong_thesis_date():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a correct thesis date')
+def test_correct_thesis_date():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a correct url')
+def test_correct_url():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Submit thesis and verify record detail in the list page')
+def test_verify_thesis_record_in_the_list_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Submit thesis and verify record detail in the detail page')
+def test_verify_thesis_record_in_the_detail_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Accept thesis and verify the confirmation message')
+def test_verify_accepted_thesis():
+    pass
+
+
+@given('I am logged in')
+def author_user(login):
```

Comment:
Now that we use BDD, can the sentence `I am logged in` become a reusable piece of code that does the login? Not to mix different styles, we can remove the fixture.